lib/jekyll-octicons.rb

```diff
@@ -4,10 +4,18 @@
 
 module Jekyll
   class Octicons < Liquid::Tag
+    # Syntax for the octicon symbol
+    Syntax = /\A(#{Liquid::VariableSignature}+)/
 
-    def initialize(tag_name, options, tokens)
+    # Copied from Liquid::TagAttributes to allow dashes in tag names:
+    #
+    #   {% octicon alert area-label:"Hello World!" %}
+    #
+    TagAttributes = /([\w-]+)\s*\:\s*(#{Liquid::QuotedFragment})/o
```

Comment:
@jonrohan I just copied the regex from liquid to allow dashes in attribute names.